tests/install_upgrade_operators/crypto_policy/test_pqc_tls_audit.py

```
@@ -0,0 +1,91 @@
+    "kubevirt-migration-prometheus": "CNV-83219",
+}
+
+
+class TestPqcNodeCapability:
+    @pytest.mark.polarion("CNV-15221")
+    def test_node_openssl_supports_pqc_groups(
+        self,
+        node_available_tls_groups,
+    ):
+        """Verify that worker node OpenSSL supports post-quantum TLS groups."""
+        LOGGER.info(f"Available TLS groups on node: {node_available_tls_groups}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)